NYPL-discovery/discovery-front-end
# SCC-2460: Subject Heading Search with react autosuggest

**What's this do?**
Implements `react-autosuggest` as used by Design System for subject heading search.

**Why are we doing this? (w/ JIRA link if applicable)**
https://jira.nypl.org/browse/SCC-2460

**Do these changes have automated tests?**
Added tests that check for the existence. It is an external library. So not sure how much this needs to be tested

**How should this be QAed?**
Check that the search works correctly. It functions a little different than the original when using the up and down arrows. This can maybe be manipulated. So will ask for Sean's input on that.

**Dependencies for merging? Releasing to production?**
[Description of any watchouts, dependencies, or issues we should be aware of]

**Has the application documentation been updated for these changes?**

**Did someone actually run this code to verify it works?**
I did
